packages/studio-ui/tsconfig.json

```diff
@@ -21,5 +21,5 @@
     }
   },
   "include": ["src/**/*.ts", "src/**/*.tsx"],
-  "exclude": ["node_modules", "build", "dist", "example", "rollup.config.js", "webpack.config.js", "**/*.test.ts"]
```

Comment:
This list of paths excluded was unnecessary as we only include files inside the src dir.